### PR TITLE
Implement update attributes for objects

### PR DESCRIPTION
**Pull request by baloo**

### CODE REVIEW

**Comment by wiktor-k** [comment]
Ugh, the build failed due to #114 not being merged :-/

**Comment by baloo** [comment]
I'll rebase when that merges :)

**Review by ionut-arm — APPROVED** [review]
👍🏻 Thanks!

**Comment by wiktor-k** [comment]
I've replaced #114 with #122 that has been merged so you're good to go for rebasing this PR :shipit:

**Review by wiktor-k — APPROVED** [review]
LGTM 👍 thanks!

File: cryptoki/tests/basic.rs [hunk 1]
```diff
@@ -826,3 +826,46 @@ fn aes_cbc_pad_encrypt() -> TestResult {
     assert_eq!(expected_cipher[..], cipher[..]);
     Ok(())
 }
+
+#[test]
+#[serial]
+fn update_attributes_key() -> Result<()> {
```
Comment (wiktor-k) [inline cryptoki/tests/basic.rs:832]: This should be `TestResult` now I think.

Comment (baloo) [reply cryptoki/tests/basic.rs:832]: sorry, missed that.

**Comment by wiktor-k** [comment]
@baloo Sorry for the annoyance but could I bother you with squashing these two commits? :pray:  Maybe it's minor but it's good for `git bisect` if each commit works and passes the CI.

After that I'll make sure to merge it ASAP :)

**Comment by wiktor-k** [comment]
Thanks for everything! 👋

**Pull request merged**